website/static/js/profile.js

```diff
@@ -741,6 +744,17 @@ $.extend(SocialViewModel.prototype, SerializeMixin.prototype, TrackedMixin.proto
 
 SocialViewModel.prototype.serialize = function() {
     var serializedData = ko.toJS(this);
+    // Do some extra work to store these as arrays in the DB
+    var arrayInDBKeys = ['twitter', 'linkedIn', 'github'];
+    $.each(serializedData || {}, function(key, value) {
```

Comment:
Same here: more straightforward to loop over the arrayInDBKeys than the whole input dictionary.

Reply:
Also, a nit: we no longer need to use `$.each`, since--as I learned yesterday--we're not supporting browsers older than IE 11, which means `Array.prototype.forEach` is AOK.